application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarEventSheet.xml

```diff
@@ -117,6 +117,7 @@ $doc.display($field)
   #end
 #end
 {{/velocity}}{{velocity}}{{html wiki="true" clean="false"}}
+#template('display_macros.vm')## Comment to escape new line so the UI displays properly
```

Comment:
The comment is not actually needed, since it means we should add this comment every time we use `##` like this